plugins/woocommerce-admin/client/marketplace/components/search-results/search-results.tsx

```diff
@@ -19,15 +34,46 @@ export default function SearchResults( props: SearchResultProps ): JSX.Element {
 		( product ) => product.type === ProductType.theme
 	);
 
+	function navigateToTab( tab: ProductType ) {
+		navigateTo( {
+			url: getNewPath( {
+				category: ALL_CATEGORIES_SLUGS[ tab ],
+				tab: TAB[ tab ],
```

Comment:
Not sure if this is a good idea, but what if we keep the tab as search, but add another prop like `section`.
This would simplify the code in `Tabs` and remove the need for having `ALL_CATEGORIES_SLUGS`.

The `section` could just show or hide parts of `SearchResults`.

Reply:
This is definitely a better approach, though I think both could work. I've refactored to do things this way in 6263b0025b807d0d3ea3089b0cb774bad59f61d2